### PR TITLE
PDF Template Updates

### PR DESCRIPTION
**Pull request by anthonyvelazquez**
# PDF Template Updates 

### What was the problem?

- Some headings needed to be a larger font size
- Some timestamps were missing time zones
- Some text needed to be on its own line

### CODE REVIEW

File: service-python/pdfgenerator/src/requirements.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@ pdfkit==1.0.0
 pytest==7.1.2
 pytest-cov==3.0.0
 python-dateutil==2.8.2
+pytz
```
Comment (jbregstone) [inline service-python/pdfgenerator/src/requirements.txt:15]: You should specify a version

**Review by jbregstone — CHANGES_REQUESTED** [review]
The sample data on the swagger page returns an error. I think the medication data may not be parse-able.

**Comment by anthonyvelazquez** [comment]
> The sample data on the swagger page returns an error. I think the medication data may not be parse-able.
> 
> edit: the medication authoredOn, in the sample request data, is not matching the expected format.

Swagger Example is fixed

File: service-python/pdfgenerator/src/lib/templates/hypertension/rating_schedule.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<h3>Hypertension Rating Schedule</h3>
+<h2>Hypertension Rating Schedule</h2>
```
Comment (jbregstone) [inline service-python/pdfgenerator/src/lib/templates/hypertension/rating_schedule.html:1]: This change does not seem to have an effect on the text size in the PDF. I tried putting in the <h1> or <h6> and it looks the same.

Comment (anthonyvelazquez) [reply service-python/pdfgenerator/src/lib/templates/hypertension/rating_schedule.html:1]: Hmm not sure what the issue might be. I reran it after making the same updates and got the changes
image
image
image

Comment (jbregstone) [reply service-python/pdfgenerator/src/lib/templates/hypertension/rating_schedule.html:1]: Interesting, I tried rebuilding a couple times but I will try again

Comment (jbregstone) [reply service-python/pdfgenerator/src/lib/templates/hypertension/rating_schedule.html:1]: Screen Shot 2022-09-12 at 1 46 54 PM
For me it still looks like this, not sure why

**Review by jbregstone — APPROVED** [review]

**Pull request merged**